wp/resources/sample_binaries/cbat-multicompiler-samples/csmith/run_wp.sh

```diff
@@ -19,4 +19,4 @@ run () {
     --wp-file2=csmith-16812.bpj
 }
 
-compile && run
+run
```

Comment:
I'm wondering if we should remove the
```
compile () {
  make
}
```
on these scripts since we won't be using it anymore. What are your thoughts?

Reply:
Yes, that makes sense! +1 I'll remove them